Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -0,0 +1,73 @@
+require "formula"
+
+module Homebrew
+  def bump_formula_pr
+    formula_arg = ARGV.named.first
+    odie "No formula specified!" unless formula_arg
+    formula = Formula[ARGV.named.first]
+    odie "No #{formula} formula found!" unless formula
```

Comment:
I've got a feeling and tried that and it didn't work for this case but I'll try it again.